src/components/Pages/LoginPage.tsx

```diff
@@ -58,31 +80,110 @@ const Button = styled.button`
 	align-items: center;
 `;
 
-const LoginPage = () => {
+const LoginPage = (props: any) => {
 	const navigate = useNavigate();
 	const RegisterNavigate = () => {
 		navigate('/Register');
 	};
 
-	const handleSubmit = (event) => {
-		event.preventDefault();
+	const {
+		register,
+		handleSubmit,
+		setError,
+		formState: { errors },
+		reset,
+	} = useForm({
+		resolver: yupResolver(schema),
+	});
+
+	const onSubmit = (data) => {
+		console.log(data);
+		reset();
 	};
 
-	return (
-		<Container>
-			<Wrapper>
-				<Title>LOGIN</Title>
-				<Form onSubmit={handleSubmit}>
-					<Input placeholder='Username'></Input>
-					<Input placeholder='Password'></Input>
+	const styles = {
+		input: {
+			color: 'white',
+		},
+	};
 
-					<Button> LOGIN </Button>
-				</Form>
-				<Register onClick={RegisterNavigate}>
-					Don't have an account? Sign Up!
-				</Register>
-			</Wrapper>
-		</Container>
+	const { classes } = props;
+
+	const theme = createTheme();
+
+	return (
+		<ThemeProvider theme={theme}>
+			<Containers>
+				<Container component='main' maxWidth='xs'>
+					<CssBaseline />
+					<Box
+						sx={{
+							marginTop: 8,
+							display: 'flex',
+							flexDirection: 'column',
+							alignItems: 'center',
+							bgcolor: 'white',
+							width: '400px',
+						}}>
+						<Avatar sx={{ m: 1, bgcolor: 'primary.main' }}>
+							<LockOutlinedIcon />
+						</Avatar>
+						<Typography component='h1' variant='h4'>
+							Sign in
+						</Typography>
+						<Box
+							onSubmit={handleSubmit(onSubmit)}
+							component='form'
+							noValidate
+							sx={{ mt: 1 }}>
+							<TextField
+								{...register('email', { required: true })}
+								margin='normal'
+								required
+								fullWidth
+								id='email'
+								label='Email Address'
+								name='email'
+								autoComplete='email'
+								autoFocus
+								InputProps={{ inputProps: { style: { color: '#fff' } } }}
+							/>
+							{errors.email?.message}
+							<TextField
+								{...register('password', { required: true })}
+								margin='normal'
+								required
+								fullWidth
+								name='password'
+								label='Password'
+								type='password'
+								id='password'
+								autoComplete='current-password'
+							/>
```

Comment:
create a common component that takes
1) error message via props
2) component via children
const TextFieldWithError = ({errorMessage, children}) => (<>{errorMessage}{children}</>)